Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/keysubmission/AnalyticsKeySubmissionStorage.kt

```diff
@@ -6,78 +6,105 @@ import de.rki.coronawarnapp.util.preferences.clearAndNotify
 import de.rki.coronawarnapp.util.preferences.createFlowPreference
 import javax.inject.Inject
 
-class AnalyticsKeySubmissionStorage @Inject constructor(
-    @AppContext val context: Context
+class AnalyticsPCRKeySubmissionStorage @Inject constructor(
+    @AppContext context: Context
+) : AnalyticsKeySubmissionStorage(context, "") // the original
+
+class AnalyticsRAKeySubmissionStorage @Inject constructor(
+    @AppContext context: Context
+) : AnalyticsKeySubmissionStorage(context, "_RAT")
```

Comment:
Should we add a little test for this?

Reply:
Will add unit tests in a separate PR. I tested it manually